### ISSUE TITLE
validateSchema validating incorrectly to true

### ISSUE THREAD

**Issue by simon-p-r** [zaggino/z-schema#143]
Hi

I have this schema which is being validated to true, here is a sample of a schema that passes validation.  I expect it to be false due to format custom not being a registered format.

This doens't fail
````js

{
  "metaSchema": {
    "name": "department",
    "type": "record",
    "version": 1,
    "jsonSchema": "v4",
    "base": "entity"
  },
  "schema": {
    "id": "department",
    "type": "object",
    "properties": {
      "recType": {
        "type": "string",
        "format": "custom"
      }
   },
   "required": ["recType"]     
  }
}
```
Whilst this one does?
````js
 {
    "id": "department",
    "type": "object",
    "properties": {
      "recType": {
        "type": "string",
        "format": "custom"
      }
   },
   "required": ["recType"]     
  }

**Comment by zaggino** [comment]
Can you please submit a failing test?

**Comment by simon-p-r** [comment]
Closed as per #149
